### DISCUSSION TITLE
Week 13 征集

### DISCUSSION THREAD

**Discussion by rewired-gh** [hitszlug/weekly#58] [Ideas]
速速来点内容！

**Comment by LittleYe233** [comment]
## 今天看了啥

![image](https://user-images.githubusercontent.com/30514318/176691515-cb11e8dd-1879-412c-907f-304cf5cee269.png)

访问的 URL 为 http://\<ip\>:6000 ，被 Firefox 强制阻止。具体原因和 Workaround 见 <https://www-archive.mozilla.org/projects/netlib/portbanning> 。

PS：6000 被标记为 X11 端口，默认被阻止。

PS2：按照上面的 Workaround 改了没用。遂更换 Web APP 端口。

**Reply by rewired-gh** [reply]
The link is broken.

**Reply by LittleYe233** [reply]
> The link is broken.

看来github自动把句号纳入 URL 了。已修改。

**Reply by rewired-gh** [reply]
![](https://n.hopp.top/lgtm.svg)
